### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [cosmoart/Freesets#5] [Ideas]
I forked the project and was planning on doing a PR with the new resources I have. But I run into an issue which I believe its because I don't have the env variables. (Error: supabaseUrl is required.) - If someone could guide me on how to fix that, I will be happy to make some changes and do the PR.

**Comment by cosmoart** [comment]
Previously, the database and images were stored directly in the repository, allowing pull requests with new resources. However, as the project grew larger, I moved the database and images to achieve better organization and reduce the repository's size.

Currently, the only way to contribute resources is by suggesting them in an Issue or Discussion, and I will add them manually.

I would love for people to contribute. I was thinking of adding a form to the project where people can suggest new resources or bring back the databases and images to the repository. Although I'm not convinced by any solution yet.

**Comment by JuanPabloDiaz** [comment]
Sure! You can make it easier for contributors to suggest resources by setting up `issue templates` in your repository. This way, you can ask for the information you need from contributors and make sure their suggestions match your project’s standards. Contributors will fill out the template fields when they open an issue, giving details like resource name, URL, description, and tags. This method ensures uniformity and helps you collect the relevant information.

I could also suggest a couple of `ISSUE TEMPLATES` that might work

**Comment by cosmoart** [comment]
I decided to move the database to the repository, but not the images, now everyone is welcome to contribute 🎉
